### PR TITLE
Clarify how to edit PHP config

### PR DESCRIPTION
**Pull request by lolautruche**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -34,11 +34,21 @@ functions:
...
 ```ini
-;php/conf.d/php.ini
+;php/conf.d/amqp.ini
+; Loading the AMQP extension.
 extension=/opt/bref-extra/amqp.so
 ```
 
+```ini
+;php/conf.d/php.ini
+memory_limit=128M
```
Comment (Nyholm) [inline Readme.md:49]: This is not really needed, right?

File: Readme.md [hunk 2]
```diff
@@ -68,20 +78,26 @@ This version of imagick is built against newer version of ImageMagick than the b
 
 ### Blackfire installation
 
-The Blackfire layer only have probe installed. You still need to install the agent. 
-The agent is installed on a separate server (not a lambda function). The smallest 
+The Blackfire layer only have probe installed.
```
Comment (Nyholm) [inline Readme.md:81]:
```suggestion
The Blackfire layer only have the probe installed. 
```

??

**Review by Nyholm — APPROVED** [review]
Thank you

File: Readme.md (hunk 1 above)
Comment (lolautruche) [reply Readme.md:49]: It shows that one can also modify any PHP setting

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**